### PR TITLE
Allow custom tags to be sourced from environment variables; update un…

### PR DESCRIPTION
**Pull request by hanwavefront**
…it tests

### CODE REVIEW

File: src/main/java/com/wavefront/opentracing/TracerParameters.java [hunk 1]
```diff
@@ -104,18 +106,47 @@ static Integer toInteger(String value) {
...
-    if (components.length % 2 == 0) {
-      Map<String, String> customTags = new HashMap<>();
-      for (int i = 0; i < components.length; i += 2) {
-        customTags.put(components[i], components[i + 1]);
-      }
-      return customTags;
-    } else {
-      logger.log(Level.WARNING, "Failed to convert Tracer parameter value '" + value +
-          "' to custom tags -- key exists without value");
+  static Map<String, String> toCustomTags(Map<String, String> params) {
```
Comment (sushantdewan123) [inline src/main/java/com/wavefront/opentracing/TracerParameters.java]: mark the method @Nullable if you are returning null?

Comment (hanwavefront) [reply src/main/java/com/wavefront/opentracing/TracerParameters.java]: fixed

File: pom.xml [hunk 2]
```diff
@@ -26,9 +26,15 @@
...
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <version>5.5.2</version>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```
Comment (vikramraman) [inline pom.xml]: As mentioned offline, let's remove this dependency.

Comment (hanwavefront) [reply pom.xml]: Removed and replaced with mock of `getEnvVariable()` using PowerMock

**Pull request merged**